pkg/controller/install/webhook.go

```diff
@@ -162,6 +182,71 @@ func (i *StrategyDeploymentInstaller) createOrUpdateValidatingWebhook(ogNamespac
 			log.Warnf("could not update ValidatingWebhookConfiguration %s", webhook.GetName())
 			return err
 		}
+
+		if len(desc.ConversionCrds) != 0 {
+			if err := createOrUpdateConversionCrds(desc, webhook.Webhooks[0].ClientConfig, i); err != nil {
+				return err
+			}
+		}
+	}
+
+	return nil
+}
+
+func createOrUpdateConversionCrds(desc v1alpha1.WebhookDescription, clientConfig admissionregistrationv1.WebhookClientConfig, i *StrategyDeploymentInstaller) error {
+
+	for index, ConversionCrd := range desc.ConversionCrds {
+		crd, err := i.strategyClient.GetOpLister().APIExtensionsV1().CustomResourceDefinitionLister().Get(desc.ConversionCrds[index])
+
+		// check if CRD is present on cluster
+		if err != nil {
+			log.Info("CRD not found %s, error: %s", desc.ConversionCrds[index], err.Error())
+		}
+
+		log.Info("Found CRD %s", desc.ConversionCrds[index])
+
+		// check if the CSV owns this CRD
+		foundCRD := false
+		csv, err := i.strategyClient.GetOpLister().OperatorsV1alpha1().ClusterServiceVersionLister().ClusterServiceVersions(i.owner.GetNamespace()).Get(i.owner.GetName())
+		ownedCRDs := csv.Spec.CustomResourceDefinitions.Owned
+		for _, ownedCRD := range ownedCRDs {
+			if ownedCRD.Name == crd.GetName() {
+				log.Info("CSV %q owns CRD %q", csv.GetName(), crd.GetName())
+				foundCRD = true
+			}
+		}
+
```

Comment:
Is it true that if the CSV does not own this CRD then we can return early? It looks like the conversion strategy logic below is only applied if `foundCRD` is true. 

So maybe we can add 
```
if !foundCRD { 
     log.Info("CSV does not own CRD...") 
     return nil 
} 
```
here and simplify the logic below

Reply:
Yep, this is a good point, thanks. I will use this to simplify the logic.